jerry-core/parser/js/opcodes-dumper.h

```diff
@@ -24,511 +24,28 @@
 #include "rcs-records.h"
 #include "scopes-tree.h"
 
-/**
- * Operand (descriptor of value or reference in context of parser)
- */
-class jsp_operand_t
+typedef enum type_t : uint8_t
```

Comment:
By the way, `: type_name` after `enum` is also a C++ feature. Seems that this could be replaced with `packed` attribute, but I'm not currently sure that it is portable between various toolchains.